api/client/webclient/webconfig.go

```diff
@@ -55,6 +55,8 @@ type WebConfig struct {
 	TunnelPublicAddress string `json:"tunnelPublicAddress,omitempty"`
 	// RecoveryCodesEnabled is a flag that determines if recovery codes are enabled in the cluster.
 	RecoveryCodesEnabled bool `json:"recoveryCodesEnabled,omitempty"`
+	// IsDashboard is a flag that determines if the cluster is running as a "dashboard".
```

Comment:
I don't know that people will understand what this means. More context would be helpful.

```suggestion
	// IsDashboard is a flag that determines if the cluster is running
	// as a "dashboard". The web UI for dashboards provides functionality
	// for downloading self-hosted licenses and other ...
```